docs/migration-guides/v7.0.0.md

```diff
@@ -1,14 +1,40 @@
 # v7.0.0 Migration guide
 
+- [Linting](#linting)
+  - [`.eslintrc`](#eslintrc)
+  - [`.eslintignore`](#eslintignore)
+- [Rendering Strategy](#rendering-strategy)
+  - [Static apps](#static-apps)
+    - [Rendering](#rendering)
+    - [`locales`](#locales)
+    - [`env`](#env)
+  - [Server rendered apps](#server-rendered-apps)
+    - [`port`](#port)
+    - [Server entry](#server-entry)
+
+## Linting
+
+### `.eslintrc`
+
+If you have previously added an [`.eslintrc`](https://eslint.org/docs/user-guide/configuring) file to your project, it is now generated and managed by sku. It's also added to your `.gitignore` automatically. This ensures it&rsquo;s always consistent with the `sku lint` command, while supporting editor integrations to give you live ESLint feedback during development.
+
+If your existing `.eslintrc` file contains some customisations, these will now need to be provided via a `dangerouslySetESLintConfig` function in `sku.config.js`.
```

Comment:
Worth calling out that consumers need to physically remove an existing `.eslintrc` from version control. We covered this in the [migration guide for v6](https://github.com/seek-oss/sku/releases/tag/v6.0.0)